packages/defaults.yaml

```diff
@@ -21,3 +21,12 @@ packages:
     required:
       states: []
       pkgs: []
+  snaps:
+    package: snapd
+    collides: ['snap',]
```

Comment:
In Debian (and probably Ubuntu) is a gene manipulation software, not related to snap packages? Is this correct?

Reply:
On Fedora, snapd was conflicting with snap package?  https://admin.fedoraproject.org/pkgdb/package/rpms/snap/

Reply:
Oh, I see. Debian's https://packages.debian.org/stretch/snap :smile:

Reply:
Ha Ha.   I thought your reference to _gene manipulation_ was an allegory to something ...

Reply:
hehe, no, it's a 'real thing' :smile:

`packages.snaps.collides` seems a candidate for `osfamilymap.yml` or `osmap.yml`, with an empty default, right?

Reply:
Does `Default` get matched in osmap.yaml?  I was grepping inside a bunch of salt formulas trying to find example of usage but didn't and not sure.  But if it is supported, absolutely yes.

Reply:
Check the [map.jinja](https://github.com/saltstack-formulas/packages-formula/blob/master/packages/map.jinja) file: the order is to set defaults from defaults.yml (so 'yes', to your question), override with osfamilymap, then with osmap, then with pillar. Unless something's wrong with the `map.jinja` file, which is always possible :smile:

Reply:
Yes, but I was wondering if following stanza works in osmap.yml?   Will this match any OS not already listed?

```
...

Default:
   snaps:
      collides:  ...
```

Reply:
No, not `Default`, but `Fedora`. It should be another part of the existing `Fedora` entry in [osmap.yml](https://github.com/saltstack-formulas/packages-formula/blob/master/packages/osmap.yaml), just like the existing one for `pips`. I think that will do.

Reply:
I tested on Ubuntu and no collision. Snap is not in OpenSUSE repo.  snapd is not available for Centos yet.

Looks like Fedora only issue. I have updated the commit accordingly.